crates/runtime/tests/runtime_tests.rs

```diff
@@ -3,24 +3,24 @@ use svm_sdk as sdk;
 use svm_sdk::traits::Encoder;
 use svm_sdk::ReturnData;
 
-use svm_codec::{Field, ParseError};
+use svm_codec::ParseError;
 use svm_layout::FixedLayout;
 use svm_program::ProgramError;
 use svm_runtime::{testing, Runtime, ValidateError};
 
-use svm_types::{Address, Context, Envelope, Gas, RuntimeError};
-use svm_types::{DeployReceipt, SpawnReceipt};
+use svm_types::{
+    Address, BytesPrimitive, Context, DeployReceipt, Envelope, Gas, RuntimeError, SpawnReceipt,
+};
 
 #[test]
-fn memory_runtime_validate_deploy_not_enough_bytes() {
+fn memory_runtime_validate_deploy_eof() {
     let runtime = testing::create_memory_runtime();
     let message = vec![0xFF, 0xFF];
 
-    let error = ParseError::NotEnoughBytes(Field::SectionKind);
-    let expected = ValidateError::Parse(error);
-
-    let actual = runtime.validate_deploy(&message).unwrap_err();
-    assert_eq!(expected, actual);
+    assert!(matches!(
```

Comment:
The previous version of the test had a different error, it said what field was missing bytes

Reply:
As agreed in chat, this is good enough considering that `Codec::decode` only deals with auto-generated inputs.